test-data/unit/fine-grained.test

```diff
@@ -1717,6 +1717,8 @@ N = NamedTuple('N', [('x', int)])
 [file a.py]
 def f() -> None: pass
 [file a.py.2]
+[builtins fixtures/tuple.pyi]
+[builtins fixtures/tuple.pyi]
```

Comment:
Why this appears twice here and in a couple dozen other tests below?

Reply:
I probably ran the script that added these twice for this file, or I did some these manually and the script replicated them.